core/lib/runner.js

```diff
@@ -400,6 +400,15 @@ function runScenario(script, intermediate, runState, contextVars) {
       script.scenarios[i].name,
       script.scenarios[i].weight);
 
+  if(script.scenarios[i].apiVersions){
+    contextVars ?
```

Comment:
if don't exist either:
1. ignore and don't add apiVersions to the object - doesn't sound right
2. inititalize the object and assign the value to it - only concern is can it ruin something in the flow after?